examples/clipboard/Clipboard.cpp

```diff
@@ -0,0 +1,79 @@
+
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Window/Clipboard.hpp>
+#include <SFML/Graphics.hpp>
+#include <iostream>
+
+
+////////////////////////////////////////////////////////////
+/// Entry point of application
+///
+/// \return Application exit code
+///
+////////////////////////////////////////////////////////////
+int main()
+{
+    // Create the window of the application
+    sf::RenderWindow window(sf::VideoMode(800, 600), "SFML Clipboard");
+
+    // Load the font file
+    sf::Font font;
+
+    if(!font.loadFromFile("resources/sansation.ttf"))
+    {
+        std::cerr << "Failed to load font file." << std::endl;
+        return EXIT_FAILURE;
+    }
+
+    // Setup strings for later use
+    sf::String str_a = "Hello World!";
```

Comment:
Variable names such as `str_a` do not follow our naming conventions (should be camelCase).